mailer/template.go

```diff
@@ -4,14 +4,13 @@ import (
 	"github.com/badoux/checkmail"
 	"github.com/netlify/gotrue/conf"
 	"github.com/netlify/gotrue/models"
-	"github.com/netlify/mailme"
 )
 
 // TemplateMailer will send mail and use templates from the site for easy mail styling
 type TemplateMailer struct {
 	SiteURL string
 	Config  *conf.Configuration
-	Mailer  *mailme.Mailer
+	Mailer  *MailmeMailer
```

Comment:
we might want to patch mailme so that it can take a custom http client for fetching templates

Reply:
Do you wanna do this now or in a different PR?

Reply:
not for now